instrumentation/jetty-httpclient/jetty-httpclient-9.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/JettyHttpClient9AgentTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2;
+
+import io.opentelemetry.instrumentation.jetty.httpclient.v9_2.AbstractJettyClient9Test;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientInstrumentationExtension;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class JettyHttpClient9AgentTest extends AbstractJettyClient9Test {
```

Comment:
JUnit 5 test classes don't need to be public
```suggestion
class JettyHttpClient9AgentTest extends AbstractJettyClient9Test {
```